lib/widgets.js

```diff
@@ -203,51 +267,30 @@ exports.multipleRadio = function (options) {
     var userAttrs = getUserAttrs(opt);
     w.toHTML = function (name, field) {
         var f = field || {};
-        return Object.keys(f.choices).reduce(function (html, k) {
+        var choices = unifyChoices(f.choices, 0);
+        return renderChoices(choices, function (choice) {
             // input element
-            var id = f.id === false ? false : (f.id ? f.id + '_' + k : 'id_' + name + '_' + k);
-            var checked = f.value && (is.array(f.value) ? f.value.some(function (v) { return String(v) === String(k); }) : String(f.value) === String(k));
+            var id = f.id === false ? false : (f.id ? f.id + '_' + choice.value : 'id_' + name + '_' + choice.value);
+            var checked = isSelected(f.value, choice.value);
             var attrs = {
                 type: 'radio',
                 name: name,
                 id: id,
                 classes: w.classes,
-                value: k,
+                value: choice.value,
                 checked: !!checked
             };
             var inputHTML = tag('input', [attrs, userAttrs, w.attrs || {}]);
             // label element
-            var labelHTML = tag('label', { 'for': id, classes: w.labelClasses }, f.choices[k]);
+            var labelHTML = tag('label', { 'for': id, classes: w.labelClasses }, choice.label);
 
-            return html + inputHTML + labelHTML;
-        }, '');
+            return inputHTML + labelHTML;
+        });
     };
     return w;
 };
 
 exports.multipleSelect = function (options) {
     var opt = options || {};
-    var w = {
-        classes: opt.classes,
-        type: 'multipleSelect'
-    };
-    var userAttrs = getUserAttrs(opt);
-    w.toHTML = function (name, field) {
-        var f = field || {};
-        var optionsHTML = Object.keys(f.choices).reduce(function (html, k) {
-            var selected = f.value && (is.array(f.value) ? f.value.some(function (v) { return String(v) === String(k); }) : String(f.value) === String(k));
-            return html + tag('option', {
-                value: k,
-                selected: !!selected
-            }, f.choices[k]);
-        }, '');
-        var attrs = {
-            multiple: true,
-            name: name,
-            id: f.id === false ? false : (f.id || true),
-            classes: w.classes
-        };
-        return tag('select', [attrs, userAttrs, w.attrs || {}], optionsHTML);
-    };
-    return w;
+    return exports.select(assign(opt, { multiple: true }));
```

Comment:
do you favor one way over the other? i find the option to be a natural thing because the html select field is already "configurable" this way. the reason i kept multipleSelect is just backwards compatibility. if you don’t mind it, i’ll remove it.